web-starter/index.js

```diff
@@ -5,11 +5,7 @@ var generators = require('yeoman-generator'),
   glob = Promise.promisify(require('glob')),
   fs = require('fs'),
   pkg = require('../package.json'),
-  ygp = require('yeoman-generator-bluebird'),
-  drupal_modules = require('drupal-modules');
-
-var DRUPAL_GESSO_URL = "https://updates.drupal.org/release-history/gesso/all";
-var SASS_CHOICES = ['Libsass','Ruby Sass'];
```

Comment:
This change should be reversed since it is not part of the change to source of Gesso.